src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -867,19 +867,21 @@ Remove the '{0}' reference from your project and add the '{1}' NuGet package ins
 {0} - The missing tool name</comment>
   </data>
   <data name="XA5207" xml:space="preserve">
-    <value>Could not find android.jar for API level {0}. This means the Android SDK platform for API level {0} is not installed. Either install it in the Android SDK Manager ({2}), or change the Xamarin.Android project to target an API version that is installed. ({1} missing.)</value>
+    <value>Could not find android.jar for API level {0}. This means the Android SDK platform for API level {0} is not installed. {2} ({1} missing.)</value>
     <comment>The following are literal names and should not be translated: android.jar
 {0} - The API level name
 {1} - The expected path of the android.jar file
-{2} - The menu location in Visual Studio that can be used to launch the Android SDK Manager</comment>
-  </data>
-  <data name="XA5207_SDK_Manager_macOS" xml:space="preserve">
-    <value>Tools &gt; Open Android SDK Manager...</value>
-    <comment>This string is the location of a menu command in Visual Studio for Mac.</comment>
+{2} - The instructions to install the missing component</comment>
   </data>
   <data name="XA5207_SDK_Manager_Windows" xml:space="preserve">
-    <value>Tools &gt; Android &gt; Android SDK Manager...</value>
-    <comment>This string is the location of a menu command in Visual Studio.</comment>
+    <value>Either install it in the Android SDK Manager (Tools &gt; Android &gt; Android SDK Manager...), or change the Xamarin.Android project to target an API version that is installed.</value>
```

Comment:
```suggestion
    <value>Either install it in the Android SDK Manager (Tools &gt; Android &gt; Android SDK Manager...), or change the project to target an API version that is installed.</value>
```
Should this say how to change the target API? The problem is we kind of lost control of that, going from `net7.0-android` to `net8.0-android` implicitly goes from 33 to 34.

Reply:
This one talks about removing `<uses-sdk/>`: https://learn.microsoft.com/en-us/dotnet/maui/migration/android-projects?view=net-maui-8.0#changes-to-androidmanifestxml

However, I do think there is one case (I helped a customer in email), we don't have a doc talking about:
* I want to stay on targetSdkVersion="33" because 34 breaks something in my app
* I want to move to .NET 8

For this case, it actually works fine to put `targetSdkVersion="33"` in your `AndroidManifest.xml` and you just ignore the resulting warnings.